### DISCUSSION TITLE
从搜索引擎搜索结果打开的哔哩哔哩视频不会正常跳转。

### DISCUSSION THREAD

**Discussion by HumanMus1c** [the1812/Bilibili-Evolved#4404] [综合讨论 / General]
无脚本、未登录的情况下有时不能跳转，有时正常跳转，有时跳转到首页推荐并且悬浮播放对应视频！

**Comment by Geeyun-JY3** [comment]
没看懂，给个截图？

**Reply by HumanMus1c** [reply]
![2023 9 3](https://github.com/the1812/Bilibili-Evolved/assets/46023154/a0221880-215a-4d15-9ef4-d4f36823f9ae)

使用必应和百度搜索引擎搜索到哔哩哔哩的视频内容，点击之后只跳转到哔哩哔哩网页首页并没有跳转到相应视频！就算成功跳转到相应视频也只是以悬浮视频展示在网页首页上。在无插件的谷歌Chrome上不登陆账号测试了一下，一切正常，但是一登陆账号就出现这个问题了。

**Reply by HumanMus1c** [reply]
谷歌Chrome浏览器无脚本、未登录的情况下有时不能跳转，有时正常跳转，有时跳转到首页推荐并且悬浮播放对应视频！
![image](https://github.com/the1812/Bilibili-Evolved/assets/46023154/f3a97c27-4398-4fb3-8c41-7d96d2d16f8b)

![2023 9 3-1](https://github.com/the1812/Bilibili-Evolved/assets/46023154/3df09eb7-12a8-4991-943a-fbcff8a20ad5)

**Reply by HumanMus1c** [reply]
甚至不登陆账号也会在首页悬浮播放对应视频。

**Reply by Geeyun-JY3** [reply]
暂无法复现（跳转到首页播放那个是破站网页端内测首页搞的新功能吗），不确定是否和破站自身相关。
在搜索引擎页面打开结果的跳转应该和本脚本没关系啊……
确定排查过其他脚本或扩展吗了？

**Reply by Farconquer** [reply]
b站新版网页端特性，从搜索引擎结果打开视频会只在首页弹窗播放，如果有美化插件之类的自然会把这个弹窗去掉，最后表现就是搜索引擎结果点击只会进入b站首页

**Reply by warmy1** [reply]
有解决办法吗？这个问题也困扰我很多次了，每次不行就要换浏览器
这次忍不了了，又花费时间找到很久不登的账号上来看看，发现早就有人反映过了
刚特意测试了下，单单关闭此脚本，悬浮视频（[后缀一般带有bvid的](https://www.bilibili.com/?bvid=BV1XC4y1g7dx)）就能正常打开
关闭其他全部脚本，只开此脚本那就打不开了，说明就是脚本的问题，可能B站改了什么东西跟现有的冲突了

**Reply by warmy1** [reply]
刚特意测试了下，单单关闭此脚本，悬浮视频（[后缀一般带有bvid的](https://www.bilibili.com/?bvid=BV1XC4y1g7dx)）就能正常打开
关闭其他全部脚本，只开此脚本那就打不开了，说明就是脚本的问题，可能B站改了什么东西跟现有的冲突了

**Comment by the1812** [comment]
无法理解 b 站为什么又搞个悬浮视频...

**Comment by the1812** [comment]
这是不是必须得开启 `清爽首页` 或 `极简首页` 才会出现问题? 这两个组件会屏蔽掉原版首页, 悬浮视频的逻辑应该是依赖原版首页的

**Reply by HumanMus1c** [reply]
是的

**Comment by rolling-yoka** [comment]
之前偶然发现的这个悬浮窗 好蠢 怎么能关掉吗...

**Comment by x460034458** [comment]
更新后仍然无法正常跳转，会反复刷新

**Comment by HumanMus1c** [comment]
现在我的账号正常了，可以开着插件从搜索引擎的结果打开对应视频页面了
